### PR TITLE
Hello Microverse Project

### PR DESCRIPTION
**Pull request by adeola003**
Here are the detailed steps I followed to complete the project before requesting a pull :

1. I created a .gitignore file and added node_modules;
2. I added a linter file;
3. Created and customized readme file based on the template provided;
4. I created an HTML file with h1 and its class;
5. I added a CSS file with my favorite color;
6. I added a test file that is ignored by git and not visible here;
7. I created a license file using the provided MIT template;

### CODE REVIEW

**Comment by adeola003** [comment]
I deleted mistakenly the previous pull request while trying to change it according to your reviews so I had to start over and complete the job again. Thank you very much for your time.

File: README.md [hunk 1]
```diff
@@ -1,57 +1,177 @@
...
+
+  - [New Micronauts]
+
+- **[From where do I begin my journey with your Company?]**
+
+  - [It all starts with the filling of our registration form]
+
+  ## 📝 License <a name="license"></a>
+
+  This project is [MIT] licensed.
```
Comment (franclobo) [inline README.md:177]: - **[OPTIONAL]** Please 🙏🏽 make sure that the MIT link works as expected. We could correct this by using 👉🏽  `[MIT](license.md)` for a professional README file.

**Review by franclobo — APPROVED** [review]
Hi @adeola003, 

# STATUS: APPROVED 😃 

git-merge

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- No linter errors ✔️ 
- Correct Github flow ✔️ 
- Descriptive PR ✔️ 
- Professional README file ✔️ 
- Good use of .gitignore file 💯 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (adeola003) [reply README.md:177]: Thank you very much.

**Pull request merged**